version_info.json

```diff
@@ -1,8 +1,8 @@
 {
   "pinned-versions": {
-    "iree-compiler": "20220726.216",
-    "iree-runtime": "20220726.216",
-    "iree-tools-xla": "20220726.216",
+    "iree-compiler": "20220809.230",
```

Comment:
We're looking to promote [20220811.232](https://github.com/iree-org/iree/releases/tag/candidate-20220714.204). Can we use that instead?

Reply:
Sadly no. This is the most recent version still compatible with 0.3.15.

Reply:
Is this due to attribute changes?